### ISSUE TITLE
[mingw64] [ffmpeg] [clang] Failed to apply patch

### ISSUE THREAD

**Issue by kliffgomel** [m-ab-s/media-autobuild_suite#1761]
During build ffmpeg displays this error:

Failed to apply patch 'https://patchwork.ffmpeg.org/project/ffmpeg/patch/1594533783-28695-1-git-send-email-mypopydev@gmail.com/mbox/'
Patch without filename, ignoring. Specify an explicit filename.

But ffmpeg ыефешс build was successful with this error.

**Comment by 1480c1** [comment]
can you open mintty and run
```bash
/usr/bin/curl -sI "https://patchwork.ffmpeg.org/project/ffmpeg/patch/1594533783-28695-1-git-send-email-mypopydev@gmail.com/mbox/" | grep -Eo 'filename=.*$' | sed 's/filename=//'
```
?

locally it prints
```bash
FFmpeg-devel-lavf-srt-fix-build-fail-when-used-the-libsrt-1.4.1.patch
```

**Comment by kliffgomel** [comment]
> 
> 
> can you open mintty and run
> 
> ```shell
> /usr/bin/curl -sI "https://patchwork.ffmpeg.org/project/ffmpeg/patch/1594533783-28695-1-git-send-email-mypopydev@gmail.com/mbox/" | grep -Eo 'filename=.*$' | sed 's/filename=//'
> ```
> 
> ?
> 
> locally it prints
> 
> ```shell
> FFmpeg-devel-lavf-srt-fix-build-fail-when-used-the-libsrt-1.4.1.patch
> ```

I get FFmpeg-devel-lavf-srt-fix-build-fail-when-used-the-libsrt-1.4.1.patch

**Comment by 1480c1** [comment]
Can you rerun to make sure it wasn't something like intermittent internet issues?

**Comment by 1480c1** [comment]
Alternatively, just to make sure, do you have an ffmpeg_extra.sh or something that is applying a srt patch?

**Comment by kliffgomel** [comment]
> 
> 
> Alternatively, just to make sure, do you have an ffmpeg_extra.sh or something that is applying a srt patch?

In the /build directory I don't have a ffmpeg_extra.sh file. The ffmpeg-git directory contains only patch files:
0001-configure-deduplicate-linking-flags.patch
0001-glslang-add-MachineIndependent.patch
0001-lavc-svt_hevc-add-libsvt-hevc-encoder-wrapper.patch

**Comment by kliffgomel** [comment]
> 
> 
> Can you rerun to make sure it wasn't something like intermittent internet issues?

I don’t think so, this is another problem.

**Comment by kliffgomel** [comment]
It's a pity that the complete log file is not saved when building the project, I only have these logs.
[ab-suite.git.reset.log](https://github.com/m-ab-s/media-autobuild_suite/files/4949289/ab-suite.git.reset.log)
[ab-suite.git.update.log](https://github.com/m-ab-s/media-autobuild_suite/files/4949290/ab-suite.git.update.log)
[ab-suite.make.log](https://github.com/m-ab-s/media-autobuild_suite/files/4949291/ab-suite.make.log)
[compile.log](https://github.com/m-ab-s/media-autobuild_suite/files/4949293/compile.log)

**Comment by 1480c1** [comment]
do you happen to have srt disabled?

**Comment by kliffgomel** [comment]
This is very strange, but collecting the ffmpeg project for the tenth time, everything went without errors. And the FFmpeg-devel-lavf-srt-fix-build-fail-when-used-the-libsrt-1.4.1.patch file appeared in the ffmpeg-git directory.
I can’t imagine what exactly for this file I could have communication problems ...
